src/components/ElectionReminders/ElectionReminder.js

```diff
@@ -5,6 +5,7 @@ import { useAuth } from "../../contexts/AuthContext";
 import { completedAction } from "../../functions/UserData";
 import { LoadingWheel } from "../LoadingWheel/LoadingWheel.component";
 import "./ElectionReminder.scss";
+import { LoadingWheel } from "../LoadingWheel/LoadingWheel.component";
```

Comment:
This is causing failure to compile on my end. Duplicate import with line 6 above.

Reply:
Hmm, running into some weird compilation errors as well. 

![image](https://user-images.githubusercontent.com/11676094/166083343-54a81c5a-d75a-4e2c-9030-e67427e678c2.png)